examples/platform/silabs/wifi/sl_wifi_if.cpp

```diff
@@ -66,11 +66,9 @@ extern "C" {
 #include "sl_wifi.h"
 #include "sl_wifi_callback_framework.h"
 #include "wfx_host_events.h"
-#if SLI_SI91X_MCU_INTERFACE
 #include "sl_si91x_trng.h"
```

Comment:
why is it ok to include this now with ncp options?

Reply:
I am not sure ok is the right word, but the 917 NCP fails to compile without it.